orbit/cmd/orbit/orbit.go

```diff
@@ -760,9 +754,10 @@ func main() {
 				return fmt.Errorf("initializing client: %w", err)
 			}
 
-			// perform an initial check to see if the token
-			// has not been revoked by the server
-			if err := deviceClient.CheckToken(trw.GetCached()); err != nil {
```

Comment:
@getvictor I think this initial call and the one above are required. We described exactly what needs to happen with Lucas here: https://github.com/fleetdm/fleet/blob/main/proposals/fleet-desktop-token-rotation.md

there are some nasty race conditions otherwise. Having  said that, maybe there's a way we can get away without them? just a heads-upp

Reply:
I can give this a good look and in-depth test tomorrow with fresh eyes/mind.

Reply:
Yes, we're still doing `deviceClient.CheckToken` (unless we know for sure it expired). The doc says
> If we have a token, load and verify its validity
and I assume "load" means load it from the file, and not write it to server.